hack/windows/Install-OVS.ps1

```diff
@@ -134,50 +142,81 @@ function DownloadOVS() {
 }
 
 function InstallOVS() {
-    # unzip OVS.
-    WaitExpandFiles $OVSZip $OVSDownloadDir
-    # Copy OVS package to target dir.
-    Log "Copying OVS package from $OVSDownloadDir\openvswitch to $OVSInstallDir"
-    mv "$OVSDownloadDir\openvswitch" $OVSInstallDir
-    if (!$LocalFile) {
-        rm $OVSZip
+    param (
+        [parameter(Mandatory = $true)] [string] $OVSInstallDir
+    )
+    # Install powershell modules
+    $OVSScriptsPath = "${OVSInstallDir}\scripts"
+    CheckAndInstallScripts -OVSScriptsPath $OVSScriptsPath
+
+    # Install VC redistributables.
+    $OVSRedistDir="$mountPath\openvswitch\redist"
```

Comment:
This is an invalid reference to `$mountPath` in this script. The script itself should not make an assumption as to whether it is run from a container or from the host.

Reply:
Good catch, I would update it.